components/common/NotificationBox/useNotification.ts

```diff
@@ -0,0 +1,45 @@
+import { Event, FilterOption } from './types'
+
+export const Interaction = {
+  SALE: 'SALE',
+  OFFER: 'OFFER',
+  ACCEPTED_OFFER: 'ACCEPTED_OFFER',
+}
```

Comment:
update event type here with `SALE`, `OFFER` and `ACCEPTED_OFFER`. 
the preview site will get a query error now since these event types are not implemented by gql.
not sure if it's the final event type. any updates please leave a comment :)

Reply:
You can still query this with `offerEvents` entity

Reply:
There is no such thing as `SALE` we have `BUY`

Reply:
Loooks like proper time to write docs 🤗. 

The available interaction types are here

https://github.com/kodadot/snek/blob/main/src/model/generated/_interaction.ts

Reply:
SALE == BUY